corehq/apps/reports/analytics/sqlaccessors.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models.aggregates import Count
+
+from corehq.apps.sofabed.models import FormData
+
+
+def get_form_counts_for_user_by_date(user_ids, startdate, enddate, date_field, timezone):
```

Comment:
good call on the name. yeah that's way clearer now that it's pulled out. before the user ids were getting pulled from the domain so it already had quasi domain filtering, but i think it's better to be explicit